### ISSUE TITLE
OpenAPI JAX-RS subresource support

### ISSUE THREAD

**Issue by JWood48** [swagger-api/swagger-core#2607]
| Q                                 | A
| --------------------------------- | -------
| Bug or feature request?           |  Feature
| Which Swagger-Core version?       | 2.0.0-rc3
| Which Java version?               |  8
| Which JAX-RS framework & version? |
 
JAX-RX Sub resource support seems to be missing from swagger 2.0.0-rc3

I have two simple classes:

```java
@OpenAPIDefinition (info = 
@Info( title = "TEST", version = "0.0", description = "Test API" ) )
@Path("/swaggertest/")
public class SwaggerTestResource {

	@GET
	@Path("name")
	@Produces(MediaType.TEXT_PLAIN)
	public String getName() {
		return "SwaggerTest";
	}
	
	@Path("subresource")
	@Operation
	public SwaggerTestSubResource getSubResource() {
		return new SwaggerTestSubResource();
	}
	
}
```

and 

```java
public class SwaggerTestSubResource {

	@GET
	@Path("version")
	@Produces(MediaType.TEXT_PLAIN)
	public String getSubResourceVersion() {
		return "1.0.0";
	}
	
}
```
I would expect the sub resource GET endpoint to show up in the openapi.json document but it looks like this:
```json
{
  "openapi" : "3.0.0",
  "info" : {
    "title" : "TEST",
    "description" : "Test API",
    "version" : "0.0"
  },
  "paths" : {
    "/swaggertest/name" : {
      "get" : {
        "operationId" : "getName",
        "responses" : {
          "default" : {
            "description" : "default response",
            "content" : {
              "*/*" : {
                "schema" : {
                  "type" : "string"
                }
              }
            }
          }
        }
      }
    }
  },
  "components" : {
    "schemas" : {
      "SwaggerTestSubResource" : {
        "type" : "object",
        "properties" : {
          "subResourceVersion" : {
            "type" : "string"
          }
        }
      }
    }
  }
}
```

Where the sub resource is included as a schema but no endpoint created.

**Comment by frantuma** [comment]
Sub resource support is work in progress and will be available in oncoming release

**Comment by JWood48** [comment]
Great! Is there a roadmap or release schedule for swagger-core with OpenAPI support?

**Comment by frantuma** [comment]
Sub resources support implemented in #2615 and available in latest `2.0.0-SNAPSHOT` available on sonatype, and in oncoming release (no fixed schedule but not far). Closing this ticket, please reopen if still experiencing issues
